scripts/dev

```diff
@@ -342,14 +342,33 @@ namespace :db do
   end
 
   desc "Dump DB to file"
-  task :dump do
-    sh "pg_dump -T *.*flyway* -T *.*existing_model --data-only postgres > scripts/data/local-db.sql"
+  task :dump, [:file] do |t, args|
```

Comment:
These changes could use [documentation here](https://github.com/CMSgov/mint-app/blob/0b35902a1f707477716706b5d7bb68428b26b4eb/docs/dev_script_usage.md#L16-L17)

Ruby does allow stuff like `scripts/dev --tasks` to get some "self documenting" things, but this doc covers a bit more of the uses for folks unfamiliar with the syntax (like myself a few minutes ago) - I struggled to figure out how to actually pass the argument to this dump/seed command for a bit 😓

Reply:
that's a good shout out, thanks Clay! I'll add to the documentation file. It is the only task I know of that currently takes arguments, so it definitely is out of the ordinary here.